fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/api/biome/v1/BiomeModifications.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.biome.v1;
+
+import java.util.function.Predicate;
+
+import org.jetbrains.annotations.ApiStatus;
+
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.RegistryKey;
+import net.minecraft.world.gen.GenerationStep;
+import net.minecraft.world.gen.carver.ConfiguredCarver;
+import net.minecraft.world.gen.feature.ConfiguredFeature;
+import net.minecraft.world.gen.feature.ConfiguredStructureFeature;
+
+/**
+ * Provides an API to modify Biomes after they have been loaded and before they are used in the World.
+ *
+ * <p>Any modifications made to biomes will not be available for use in server.properties (as of 1.16.1),
+ * or the demo level.
+ *
+ * <p><b>Experimental feature</b>, may be removed or changed without further notice.
+ * Because of the volatility of world generation in Minecraft 1.16, this API is marked experimental
+ * since it is likely to change in future Minecraft versions.
+ */
+@ApiStatus.Experimental
+public final class BiomeModifications {
+	/**
+	 * The suggested order for modifiers that enrich biomes by adding to them without relying on
+	 * other information in the biome, or removing other features.
+	 *
+	 * <p><b>Examples:</b> New ores, new vegetation, new structures
+	 */
+	public static final int ORDER_ADDITIONS = 1000;
```

Comment:
I will note the fact that by multiplying by 10 for each of these values (sometimes more than once) the result is there's an exponential increase in the possible values between each of these points. It might be a better idea to just do like
```
000000000
025000000
050000000
075000000
100000000
```
So distribution is more uniform. Really though I'm guessing it doesn't really matter.

Reply:
Yeah I'll be honest, I was kinda lazy with those :-D

Reply:
I'll leave it as is though, since it really doesn't matter all that much.